website/docs/r/arc_machine_extension.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+subcategory: "Hybrid Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_arc_machine_extension"
+description: |-
+  Manages a Hybrid Compute Machine Extension.
+---
+
+# azurerm_arc_machine_extension
+
+Manages a Hybrid Compute Machine Extension.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+data "azurerm_arc_machine" "example" {
+  name                = "existing-hcmachine"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_arc_machine_extension" "example" {
+  name           = "example"
+  location       = "West Europe"
+  arc_machine_id = data.azurerm_arc_machine.example.id
+  publisher      = "Microsoft.Azure.Monitor"
+  type           = "AzureMonitorLinuxAgent"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `arc_machine_id` - (Required) The ID of the Hybrid Compute Machine Extension. Changing this forces a new Hybrid Compute Machine Extension to be created.
+
+* `location` - (Required) The Azure Region where the Hybrid Compute Machine Extension should exist. Changing this forces a new Hybrid Compute Machine Extension to be created.
+
+* `name` - (Required) The name which should be used for this Hybrid Compute Machine Extension. Changing this forces a new Hybrid Compute Machine Extension to be created.
+
+* `publisher` - (Required) The name of the extension handler publisher, such as `Microsoft.Azure.Monitor`. Changing this forces a new Hybrid Compute Machine Extension to be created.
+
+* `type` - (Required) Specifies the type of the extension. For example `CustomScriptExtension` or `AzureMonitorLinuxAgent`. Changing this forces a new Hybrid Compute Machine Extension to be created.
+
+---
+
+* `automatic_upgrade_enabled` - (Optional) Indicates whether the extension should be automatically upgraded by the platform if there is a newer version available. Supported values are `true` and `false`.
```

Comment:
grammer
```suggestion
* `automatic_upgrades_enabled` - (Optional) Indicates whether the extension should be automatically upgraded by the platform if there is a newer version available. Supported values are `true` and `false`.
```

Reply:
@katbyte I'd love to change the property name to suggested. But I found similar property named `automatic_upgrade_enabled` in [azurerm_virtual_machine_extension](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_machine_extension#automatic_upgrade_enabled). Which one should I use?

Reply:
ah lets keep it wrong for consistency then